runtimes/runtimes/operational-telemetry/operational-telemetry-service.ts

```diff
@@ -0,0 +1,126 @@
+import { PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics'
+import { AwsMetricExporter } from './aws-metrics-exporter'
+import { OperationalTelemetry } from './operational-telemetry'
+import opentelemetry, { diag, Attributes, DiagLogLevel, trace } from '@opentelemetry/api'
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { Resource } from '@opentelemetry/resources'
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions'
+import { randomUUID } from 'crypto'
+import { RemoteConsole } from 'vscode-languageserver'
+import { AwsCognitoApiGatewaySender } from './aws-cognito-gateway-sender'
+import { AwsSpanExporter } from './aws-spans-exporter'
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
+
+export class OperationalTelemetryService implements OperationalTelemetry {
+    private customAttributes: Record<string, any> = {}
+    private static instance: OperationalTelemetryService
+    private readonly SCOPE_NAME = 'language-server-runtimes'
```

Comment:
Scope should be configurable, not hardcoded. This service should be reusable with other scopes.